.github/workflows/migration.yml

```diff
@@ -0,0 +1,46 @@
+name: "Test migrations"
+
+on:  
+  pull_request:
+    branches:
+      - "**"
+
+env:
+  DJANGO_SETTINGS_MODULE: backend.settings.localdev
+
+jobs:
+  test-migration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: main
+      - name: Set up python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v3.0.8
+        id: cache
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade --upgrade-strategy eager -r backend/dev-requirements.txt
+      - name: Run initial migration and feed data
+        run: |
+          make db
+          cd backend
+          sleep 10
```

Comment:
Is a sleep enough ? Is there another way to check if the service is ready that would "never fail".

Reply:
Best would be to wait for the db to be ready before executing the migration. However I have tested such procedure and the command was never returning and therefore the action idling. Hence the hardcoded sleep value

Reply:
Solved by using a proper service for db (like we already do for unit test).